treeherder/webapp/api/performance_data.py

```diff
@@ -763,10 +763,16 @@ def list(self, request):
         new_rev = query_params.validated_data['new_revision']
         base_repo_name = query_params.validated_data['base_repository']
         new_repo_name = query_params.validated_data['new_repository']
-        interval = query_params.validated_data['interval']
```

Comment:
Let's keep the interval field because it's needed for when we compare without a base, as we need it to send a selected timerange from the PerfCompare UI (last day or last 2 / 7 / 14 / 30 / 60 / 90 days).